grails-app/controllers/org/pih/warehouse/RoleInterceptor.groovy

```diff
@@ -13,7 +13,10 @@ import org.pih.warehouse.core.RoleType
  * */
 class RoleInterceptor {
     def userService
-    def dependsOn = [SecurityInterceptor]
```

Comment:
`dependsOn` no longer works in Grails 3. the `order` declaration below is its replacement